cashctrl_ledger/tests/test_ledger.py

```diff
@@ -97,7 +97,7 @@ def test_update_ledger_with_illegal_attributes(self, ledger):
             ledger.modify_ledger_entry(target)
 
         # Delete the ledger entry created above
-        ledger.delete_ledger_entry(id)
+        ledger.delete_ledger_entries([str(id)])
```

Comment:
Let's not require the id to be a string. Please change to:
```
ledger.delete_ledger_entries([id])
```
instead of:
```
ledger.delete_ledger_entries([str(id)])
```